web3/providers/ipc.py

```diff
@@ -154,7 +154,7 @@ def __init__(
         self.timeout = timeout
         self._lock = threading.Lock()
         self._socket = PersistantSocket(self.ipc_path)
-        super().__init__()
+        super().__init__(**kwargs)
```

Comment:
Should we pass `*args` here too? Or maybe remove it from the signature if not?

Reply:
I think removing from signature is probably the best route. I don't think we should define any args in the base classes since it would be hard to envision arg positioning to pass along as a user. Best to more explicitly set certain properties on the class imo. Thoughts on that?

Reply:
Yeah, removing it makes sense to me!